### ISSUE TITLE
[v12] PHONE ICON  error

### ISSUE THREAD

**Issue by cesarsc79** [OCA/connector-telephony#212]
When i click the phone icon:

in red:
Calling party number not retreived from IPBX or IPBX unreachable by Odoo

somebody know if this is an error?

![Captura](https://user-images.githubusercontent.com/25187665/78978871-f6e83280-7b1a-11ea-8049-a5f5d28f02a4.JPG)

**Comment by moblesmarti** [comment]
Hello, 
I have the same error, have you been able to solve it?

**Comment by integraticdev** [comment]
Hello,
I have the same error, have you been able to solve it? thanks

**Comment by ymoreira16** [comment]
Subject: PHONE ICON error: IPBX error – Calling party number not retrieved from IPBX or IPBX unreachable by Odoo

Hi everyone,

I'm getting the following error when clicking on the phone icon in Odoo:

IPBX error
Calling party number not retrieved from IPBX or IPBX unreachable by Odoo
The connection to the Asterisk server works fine — the connection test shows "Connection successful", and my Asterisk server is reachable via ARI. I already have the extension and channel configured in the user settings, and the server is correctly defined.

However, when I try to make a call via the phone icon, this error appears and nothing happens. I saw this issue mentioned before but it was closed without a solution.

Has anyone been able to solve this? Any help or insight would be greatly appreciated.

Thanks!
